### PR TITLE
Feat/adapt uds aux to dt interface

### PR DESCRIPTION
**Pull request by BKaDamien**
Adapt UDS auxiliary and server to the dt aux interface

### CODE REVIEW

File: src/pykiso/lib/auxiliaries/udsaux/common/uds_base_auxiliary.py [hunk 1]
```diff
@@ -56,91 +77,78 @@ def __init__(
...
+        self, tp_layer: Optional[dict] = None, uds_layer: Optional[dict] = None
+    ) -> None:
+        """Initialize isotp and uds layer attributes.
+
+        :param tp_layer: isotp configurration given at yaml level
+        :param uds_layer: uds configurration given at yaml level
+        """
+        tp_layer = tp_layer or UdsBaseAuxiliary.DEFAULT_TP_CONFIG
+        uds_layer = uds_layer or UdsBaseAuxiliary.DEFAULT_UDS_CONFIG
+        # add configured reauest id and response id to tp layer config
```
Comment (sebclrsn) [inline src/pykiso/lib/auxiliaries/udsaux/common/uds_base_auxiliary.py:116]:
```suggestion
        # add configured request id and response id to tp layer config
```

File: src/pykiso/lib/auxiliaries/udsaux/common/uds_base_auxiliary.py [hunk 2]
```diff
@@ -56,91 +77,78 @@ def __init__(
...
+            log.warning(
+                f"apply following values for UDS layer configuration : {UdsBaseAuxiliary.DEFAULT_UDS_CONFIG}"
+            )
+
+    def _init_com_layers(
+        self, tp_layer: Optional[dict] = None, uds_layer: Optional[dict] = None
+    ) -> None:
+        """Initialize isotp and uds layer attributes.
+
+        :param tp_layer: isotp configurration given at yaml level
```
Comment (sebclrsn) [inline src/pykiso/lib/auxiliaries/udsaux/common/uds_base_auxiliary.py:111]:
```suggestion
        :param tp_layer: isotp configuration given at yaml level
```

File: src/pykiso/lib/auxiliaries/udsaux/common/uds_base_auxiliary.py [hunk 3]
```diff
@@ -56,91 +77,78 @@ def __init__(
...
+        :param uds_layer: uds configurration given at yaml level
```
Comment (sebclrsn) [inline src/pykiso/lib/auxiliaries/udsaux/common/uds_base_auxiliary.py:112]:
```suggestion
        :param uds_layer: uds configuration given at yaml level
```

**Review by sebclrsn — CHANGES_REQUESTED** [review]

File: docs/whats_new/version_ongoing.rst [hunk 4]
```diff
@@ -72,3 +75,15 @@ Instead of having only the 2 tags "variant" and "branch_level" to select tests,
 can now set any tagname.
 
 See: :ref:`define_test_information` for more details.
+
+Lightweight UDS auxliary configuration
```
Comment (sebclrsn) [inline docs/whats_new/version_ongoing.rst:79]:
```suggestion
Lightweight UDS auxiliary configuration
```

File: docs/examples/uds_auxiliary.rst [hunk 5]
```diff
@@ -23,75 +21,52 @@ Find below a complete configuration example :
...
+          # transport_protocol -> CAN
+          # p2_can_client -> 5
+          # p2_can_server -> 1
+          uds_layer:
+            transport_protocol: 'CAN'
+            p2_can_client: 5
+            p2_can_server: 1
+          # tp_layer parameter is not mandatory and by default the following
+          # values will be applied:
+          # addressing_type -> NORMAL
```
Comment (sebclrsn) [inline docs/examples/uds_auxiliary.rst:45]: Maybe try to explain a bit what each parameter does, or at least the different acceptable values

**Review by sebclrsn — APPROVED** [review]

File: docs/examples/uds_auxiliary.rst (hunk 5 above)
Comment (BKaDamien) [reply docs/examples/uds_auxiliary.rst:45]: It was never explain before but yes why not

**Review by sebastianpfischer — APPROVED** [review]

**Comment by BKaDamien** [comment]
I don't understand codecov to be honest can somebody explain me what is going on?

**Pull request merged**